estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import fields, models
+from datetime import timedelta
+
+def default_availability_date(recordset):
+    return fields.Datetime.today() + timedelta(days=90)
```

Comment:
```suggestion
    return fields.Date.context_today(self) + timedelta(days=90)
```
It's better to use `context_today()` here so that it takes the user's timezone into account.